### PR TITLE
Add port tag or vlan,trunk options setting

### PR DESCRIPTION
**Pull request by chenyunchen**

### CODE REVIEW

File: messages/messages.proto [hunk 1]
```diff
@@ -55,6 +57,28 @@ message AddPortRequest {
   string ifaceName = 2;
 }
 
+message PortOptions {
+ int32 tag = 1;
```
Comment (zyfdegh) [inline messages/messages.proto]: alignment

File: messages/messages.proto [hunk 2]
```diff
@@ -55,6 +57,28 @@ message AddPortRequest {
...
+ string VLANMode = 2;
+ repeated int32 trunk = 3;
+}
+
+message SetPortRequest {
+  string ifaceName = 1;
+  PortOptions portOptions = 2;
```
Comment (c9s) [inline messages/messages.proto]: you are in the PortRequest, you don't have to prefix it.

File: server/openvswitch_handler.go [hunk 3]
```diff
@@ -48,6 +49,56 @@ func (s *server) AddPort(ctx context.Context, req *pb.AddPortRequest) (*pb.OVSRe
...
+	options := &pb.PortOptions{}
+	if portOptions.Tag != nil {
+		options.Tag = int32(*portOptions.Tag)
+	}
+	if portOptions.VLANMode != nil {
+		options.VLANMode = *portOptions.VLANMode
+	}
+	for _, t := range portOptions.Trunk {
+		options.Trunk = append(options.Trunk, int32(t))
+	}
```
Comment (c9s) [inline server/openvswitch_handler.go]: why do we copy the port options here?

Comment (c9s) [reply server/openvswitch_handler.go]: if there is a reason, could you please add a comment for it? thank you~

File: vendor/vendor.json [hunk 4]
```diff
@@ -287,10 +287,10 @@
 			"revisionTime": "2018-03-31T12:42:32Z"
 		},
 		{
-			"checksumSHA1": "BPTSX2pY4eNgqbQpbRt1V1DnkRM=",
+			"checksumSHA1": "QV1GkNjY6f6quzBVonyYp8jP3No=",
 			"path": "github.com/linkernetworks/go-openvswitch/ovs",
-			"revision": "eab982908a30d98500ba243e022247947d9ca570",
-			"revisionTime": "2018-06-28T03:23:49Z"
+			"revision": "614ef75bf4e8c6de85a736c9086f628a049011ec",
+			"revisionTime": "2018-06-28T09:23:35Z"
```
Comment (c9s) [inline vendor/vendor.json]: would be good to have a separated PR for updating the dependency. If the purpose is different. :-)

**Pull request merged**